pla/cs/sandbox-composer.texy

```diff
@@ -7,27 +7,15 @@ Jak stahovat Nette sandbox jednoduše pomocí Composeru?
 Nejprve si nainstalujeme Composer podle [návodu | doc:cs:composer#instalace-do-operacniho-systemu]. A poté pomoci něj stáhneme sandbox do adresáře `myApp`:
 
 /--
-composer create-project nette/sandbox myApp 2.0.5
-cd myApp
+composer create-project nette/sandbox myApp 2.3.4	# Verze 2.3.4
+# composer create-project nette/sandbox myApp		# Nebo vývojová (poslední) verze
 \--
 
-Pokud máte PHP 5.4 nebo vyšší, můžete rovnou spustit server příkazem:
+Přejdeme do adresáře s aplikací a v Linuxu/Unixu nastavíme práva.
 
 /--
-php -S localhost:3000 -t www
-\--
-
-A stránku `http://localhost:3000` otevřít v prohlížeči.
-
-
-Sandbox pro vývojovou verzi Nette
-===========
-
-Sandbox nainstalujeme stejným způsobem:
-
-/--
-composer create-project nette/sandbox myApp
 cd myApp
+chmod o+w -R {temp,log} # toto je potreba v Linuxu
```

Comment:
Diakritika.
Obecně jsem spíš proti propagování špatné konfigurace, chmod 777 by měla být až krajní možnost.
Koukni [sem](http://symfony.com/doc/current/book/installation.html#checking-symfony-application-configuration-and-setup) jak to doporučuje Symfony.